js/containers/buttons/CancelApprovalRequest.jsx

```diff
@@ -5,7 +5,7 @@ var Button = require('../../components/Button.jsx');
 
 const mapStateToProps = function(state) {
 	return {
-		display: state.approval.request_sent && !state.approval.approved,
+		display: !state.deployment.is_loading && state.deployment.data.date_requested && !state.deployment.approved && !state.deployment.rejected,
```

Comment:
maybe turn this into a `canCancel(deployState)` func so it's easier on the eyes and the brain